README.md

```diff
@@ -610,7 +610,7 @@ ARCHIVE - inactive projects or old academic projects that may lack soruce code
 
 ## DevOps
 - [r43ples](https://github.com/plt-tud/r43ples) - Revision Management for the Semantic Web
-- [RDFUnit](https://github.com/AKSW/RDFUnit) - RDFUnit is implemented on top of the Test-Driven Data Validation Ontology and designed to read and produce RDF that complies to that ontology only. 
+- [RDFUnit](https://github.com/AKSW/RDFUnit) - RDFUnit is unit testing framework that can be used either as a JVM (Java) RDF validation library or integrated in a CI/CD environment. RDFUnit checks if the given RDF data conforms to a set of OWL/RDFS ontologies or against SHACL constraints. 
```

Comment:
```suggestion
- [RDFUnit](https://github.com/AKSW/RDFUnit) - SHACL Unit Testing Suite for OWL/RDFS ontologies.
```

Reply:
a shorter version could be:
`RDF Unit testing and validation framework that supports RDFS, OWL and SHACL`
or just `RDF Unit testing and validation framework`